frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,471 @@
+        version={logoVersion}
+      />
+    );
+  };
+
+  return (
+    <>
+      <div className="co-operator-install-page__main">
+        <Helmet>
+          <title>View Install</title>
+        </Helmet>
+        <Bullseye>
+          <div id="operator-install-page">
+            {loading && (
+              <div>
+                Installing... <Spinner size="lg" />
+              </div>
+            )}
+            {!loading && isStatusFailed && (
+              <Alert variant="danger" isInline title="Installation Failed">
+                {status}: {(installObj as ClusterServiceVersionKind)?.status?.message || ''}
+              </Alert>
+            )}
+            {!loading && (
+              <Card>
+                <CardBody>
+                  <div className="co-operator-install-page__pkg-indicator">
+                    <div>{getCSVLogo()}</div>
+                    <div>{getIndicator()}</div>
+                  </div>
+                </CardBody>
+              </Card>
+            )}
+            {!loading && (
+              <Card>
+                <CardBody>{getInstallMessage()}</CardBody>
+              </Card>
+            )}
+          </div>
+        </Bullseye>
+      </div>
+      {!loading && isApprovalNeeded && (
+        <InstallPlanPreview obj={installObj as InstallPlanKind} hideApprovalBlock />
+      )}
+    </>
+  );
+};
+
+export const OperatorInstallPage: React.FC<OperatorInstallPageProps> = (props) => {
+  const { pkgNameWithVersion, targetNamespace } = props;
+  const namespace = targetNamespace;
+  const pkg = new URLSearchParams(window.location.search).get('pkg');
+  const installPageResources = [
+    {
+      kind: referenceForModel(ClusterServiceVersionModel),
+      namespaced: true,
+      fieldSelector: `metadata.name=${pkgNameWithVersion}`,
+      isList: true,
+      namespace,
+      prop: 'clusterServiceVersions',
+    },
+    {
+      kind: referenceForModel(SubscriptionModel),
+      namespaced: true,
+      fieldSelector: `metadata.name=${pkg}`,
+      isList: true,
+      namespace,
+      optional: true,
+      prop: 'subscription',
+    },
+    {
+      kind: referenceForModel(InstallPlanModel),
+      prop: 'installPlan',
+      namespaced: true,
+      namespace,
+      isList: true,
+      optional: true,
+    },
+    {
+      isList: true,
+      kind: referenceForModel(PackageManifestModel),
+      namespace: new URLSearchParams(window.location.search).get('catalogNamespace'),
+      fieldSelector: `metadata.name=${pkg}`,
+      selector: {
+        matchLabels: {
+          catalog: new URLSearchParams(window.location.search).get('catalog'),
+        },
+      },
+      prop: 'packageManifest',
+    },
+  ];
+
+  return (
+    <Firehose resources={installPageResources}>
+      <OperatorInstallStatusPage {...props} />
+    </Firehose>
+  );
+};
+
+export type OperatorInstallPageProps = {
+  activeNamespace?: string;
+  targetNamespace: string;
+  match?: match;
```

Comment:
I don't see where this is actually used?

Reply:
Another relic when I needed the URL, Removed.